source/client/client_worker_impl.cc

```diff
@@ -61,8 +60,13 @@ void ClientWorkerImpl::work() {
           stat->value();
     }
   }
+  // Note that benchmark_client_ is not terminated here, but in shutdownThread() below. This is to
+  // prevent it from influencing counters: the main thread still needs to be able to read the
```

Comment:
Nit: "to prevent the shutdown artifacts from influencing the test result counters."

Reply:
Done